### PR TITLE
Schedule Entry

### PR DESCRIPTION
**Pull request by e-scha**
I have added two new commands. You can list project schedule entries and execute project schedule entries.

### CODE REVIEW

File: fsdevtools-cli/src/main/java/com/espirit/moddev/cli/commands/scheduleEntry/ScheduleEntryStartCommand.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+        LOGGER.info(String.format("Found schedule %s on project %s", getScheduleEntryName(), super.getProject()));
+        LOGGER.info("Execute schedule");
+        ScheduleEntryControl scheduleEntryControl = scheduleEntry.execute();
+        if (isWaitForFinish()) {
+            LOGGER.info("Wait for the task to finish.");
+            scheduleEntryControl.awaitTermination();
+        }
+
+        return new SimpleResult<>(true);
```
Comment (hannespernpeintner) [inline fsdevtools-cli/src/main/java/com/espirit/moddev/cli/commands/scheduleEntry/ScheduleEntryStartCommand.java]: I don't think SimpleResult and true do fit here - the return should be a custom result type with at least the log output of the schedule and it's success/failure, don't you think?

Comment (e-scha) [reply fsdevtools-cli/src/main/java/com/espirit/moddev/cli/commands/scheduleEntry/ScheduleEntryStartCommand.java]: Hi, this is correct. For now i have no time to edit and test this. I will send a new pull request, if i ready.

**Comment by kafmann-e-spirit** [comment]
Hi @e-scha ,

we are currently working on a solution to start schedule entries in general - even server side schedule entries without project binding.

For this reason, this PR will be closed. 

Thanks for your contribution!

**Comment by DennisMaass** [comment]
Hey @kafmann-e-spirit,

i would be happy about this Feature. When will it be available?

**Comment by kafmann-e-spirit** [comment]
Hey @DennisMaass,

the Feature gets currently tested by our quality assurance. We hope that we can release this Feature in the next two weeks.

**Comment by DennisMaass** [comment]
Hey @kafmann-e-spirit,
Because of the missing changelog and documentation, i don't know if the Release 2.5.10 contains the here requested "schedule entry" feature. And if it is available, i don't know how to use it.

Can you please answer me these things?

**Comment by kafmann-e-spirit** [comment]
Hey @DennisMaass ,

yes, the latest Release (2.5.10) contains the "schedule entry" feature :)

See this [commit ](https://github.com/e-Spirit/FSDevTools/commit/9e3712b49e619282530293ddd51218e3284fcd84) for implementation details.

Short description:
* added command "schedule list"
* added command "schedule start"

Both commands can be used server-side and with a project name.

**Example for _schedule list:**

Serverwide: 
_fs-cli.exe -p schedule list_

Project: 
_fs-cli.exe -p "my ProjectName" schedule list_

For further information about the command syntax, please use _fs-cli.exe help schedule_

**Pull request closed without merging**